src/main/java/com/duckblade/osrs/toa/TombsOfAmascutConfig.java

```diff
@@ -394,6 +396,62 @@ default int chestAudioVolume()
 		return 100;
 	}
 
+	@ConfigSection(
+		name = "Reward Sarcophagus",
+		description = "Configuration for the reward sarcophagus.",
+		position = 700,
+		closedByDefault = true
+	)
+	String SECTION_REWARD_SARCOPHAGUS = "rewardSarcophagusSection";
+
+	@ConfigItem(
+		name = "Recolor white",
+		description = "Recolor the sarcophagus white lighting.",
+		position = 701,
+		keyName = "sarcophagusRecolorWhite",
+		section = SECTION_REWARD_SARCOPHAGUS
+	)
+	default boolean sarcophagusRecolorWhite()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		name = "Color",
+		description = "Color to replace sarcophagus white lighting with.",
+		position = 702,
+		keyName = "sarcophagusWhiteRecolor",
+		section = SECTION_REWARD_SARCOPHAGUS
+	)
+	default Color sarcophagusWhiteRecolor()
+	{
+		return Color.BLACK;
+	}
+
+	@ConfigItem(
+		name = "Recolor purple",
+		description = "Recolor the sarcophagus purple lighting.",
+		position = 703,
+		keyName = "sarcophagusRecolorPurple",
+		section = SECTION_REWARD_SARCOPHAGUS
+	)
+	default RecolorMode sarcophagusRecolorPurple()
+	{
+		return RecolorMode.OFF;
+	}
+
+	@ConfigItem(
+		name = "Color",
+		description = "Color to replace sarcophagus purple lighting with.",
+		position = 704,
+		keyName = "sarcophagusPurpleRecolor",
+		section = SECTION_REWARD_SARCOPHAGUS
+	)
+	default Color sarcophagusPurpleRecolor()
+	{
+		return Color.BLACK;
+	}
```

Comment:
If you extract the default model colours, you can remove the enable config options to just have the default values be the in-game colour instead. Would help to reduce the config density, but requires getting another purple to find that value.

Reply:
This one could also be done in a later update, doesn't need to be done for now.